### PR TITLE
Commit for scaffold and POC : #4

### LINKED ISSUE #4: [Component] POC and scaffold webapp
Creating a basic POC with a minimal backend and frontend.

- [x] Organize folder structure
- [x] Scaffold backend and write a basic POC
- [x] Scaffold frontend with nextjs

### PR DESCRIPTION
**Pull request by pinakipb2**
Closes #4

In this PR,
- TBD

### CODE REVIEW

File: web/backend/package.json [hunk 1]
```diff
@@ -0,0 +1,45 @@
+{
+  "name": "api_starter",
```
Comment (TanmoySG) [inline web/backend/package.json:2]: This name will change I think

**Review by TanmoySG — APPROVED** [review]
Looks good for starter @pinakipb2 ! You can fix the application name now or if you want you can do it later on.

File: web/backend/package.json (hunk 1 above)
Comment (pinakipb2) [reply web/backend/package.json]: Changed in a new commit. @TanmoySG

**Review by TanmoySG — APPROVED** [review]

**Comment by TanmoySG** [comment]
@pinakipb2  Merged, thanks!

**Pull request merged**